Sofascore2025/EventView.swift

```diff
@@ -0,0 +1,290 @@
+	
+	init(match: Event) {
+		self._match = match
+		super.init()
+	}
+	override func addViews() {
+		super.addViews()
+		addSubview(clock)
+		addSubview(startTime)
+		addSubview(splitLabel)
+		addSubview(homeTeamImage)
+		addSubview(awayTeamImage)
+		addSubview(homeTeamName)
+		addSubview(awayTeamName)
+		addSubview(homeTeamScore)
+		addSubview(awayTeamScore)
+	}
+
+	override func styleViews() {
+		super.styleViews()
+		backgroundColor = .white
+		
+		let formatter = DateFormatter()
+		formatter.dateFormat = "HH:mm"
+
+		startTime.text = formatter.string(from : Date(timeIntervalSince1970: TimeInterval(_match.startTimestamp)))
+		
+		homeTeamName.text = _match.homeTeam.name
+		awayTeamName.text =  _match.awayTeam.name
+		
+		if let homeScore = _match.homeScore {
+			homeTeamScore.text = String(homeScore)
+		}
+		if let awayScore = _match.awayScore {
+			awayTeamScore.text = String(awayScore)
+		}
+
+		
+		if let urlHome = _match.homeTeam.logoUrl, let url = URL(string: urlHome) {
+			loadImage(from: url, isHome: true)
+		}
+		if let urlAway = _match.awayTeam.logoUrl, let url = URL(string: urlAway) {
+			loadImage(from: url, isHome: false)
+		}
+		
+		
+//		clock styling -> only font is constant
+		clock.font = UIFont(name: "Roboto-Regular", size: 12)
+		clock.textAlignment = .center
+	
+//		startTime styling
+		startTime.font = UIFont(name: "Roboto-Regular", size: 12)
+		startTime.textColor = .gray
+		startTime.textAlignment = .center
+		
+//		splitLabel styling
+		splitLabel.image = UIImage(named: "Divider Horizontal")
+		splitLabel.contentMode = .scaleAspectFit
+		splitLabel.clipsToBounds = true
+		
+//		homeTeamImage styling
+		homeTeamImage.contentMode = .scaleAspectFit
+		homeTeamImage.clipsToBounds = true
+		
+//		awayTeamImage styling
+		awayTeamImage.contentMode = .scaleAspectFit
+		awayTeamImage.clipsToBounds = true
+		
+//		teamName styling -> only font is constant
+		homeTeamName.font = UIFont(name: "Roboto-Regular", size: 14)
+		awayTeamName.font = UIFont(name: "Roboto-Regular", size: 14)
+		
+//		score styling -> only font is constant
+		homeTeamScore.font = UIFont(name: "Roboto-Regular", size: 14)
+		awayTeamScore.font = UIFont(name: "Roboto-Regular", size: 14)
+		
+//		match didn't start yet
+		if (self._match.status == .notStarted) {
+			setupUpcomingStyle()
+		}
+		else if (self._match.status == .inProgress) {
+			let timer = (Int(Date().timeIntervalSince1970) - self._match.startTimestamp) / 60
+			clock.text = String(timer) + "'"
+			setupInProgressStyle()
+		}
+		else if (self._match.status == .halftime) {
+			setupHalftimeStyle()
+		}
+			
+		else if (self._match.status == .finished) {
+			setupFinishedStyle()
+		}
+		
+	}
+
+	override func setupConstraints() {
+		super.setupConstraints()
+		
+//		startTime constraints
+		startTime.snp.makeConstraints { current in
+			current.top.equalToSuperview().inset(10)
+			current.width.equalTo(56)
+			current.leading.equalToSuperview().inset(4)
+		}
```

Comment:
mozes ovako pisat za sve :)
```suggestion
		startTime.snp.makeConstraints {
			$0.top.equalToSuperview().inset(10)
			$0.width.equalTo(56)
			$0.leading.equalToSuperview().inset(4)
		}
```